Module-3-Javascript-Coding-Task/src/friday-13.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+
+
+// solution #1
+
+export const range = (min, max) => Array.from({ length: max - min + 1 }, (_, i) => min + i);
+
+export const getFriday13s = (startYear, endYear) => {
+
+    if (!startYear && !endYear) {
+        return null
+    }
+
+    const months = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+
+    const fridays = []
+
+    if (typeof endYear === "undefined") {
```

Comment:
Duplication of significant code block here. Wouldn't it work if you'd just omit the first condition? I'm guessing now.

Reply:
nope ,it doesnt work

Reply:
If you slightly modify the `range()` function, it works:
`export const range = (min, max) => min && max ? Array.from({ length: max - min + 1 }, (_, i) => min + i) : [min];`
Forget about it, I'm just toying with this.